### PR TITLE
Scorecard

### PR DESCRIPTION
**Pull request by mphprogrammer**

### CODE REVIEW

File: app/assets/javascripts/scorecards.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (plusjeff) [inline app/assets/javascripts/scorecards.js]: Can be removed

File: app/assets/stylesheets/scorecards.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the scorecards controller here.
```
Comment (plusjeff) [inline app/assets/stylesheets/scorecards.scss]: OK to remove

File: app/helpers/scorecards_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module ScorecardsHelper
```
Comment (plusjeff) [inline app/helpers/scorecards_helper.rb]: Can be removed

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -40,7 +40,10 @@
           </a>
           <ul class="dropdown-menu">
             <li><a href="<%= weams_path %>">Weams</a></li>
-          </ul>
+            <li><a href="<%= va_crosswalks_path %>">Crosswalk</a></li>
```
Comment (plusjeff) [inline app/views/layouts/_navbar.html.erb]: Can this be alphabetized?

**Comment by jeffmaher** [comment]
Is this data from the Department of Education's College Scorecard? If so, shouldn't it just be using the API instead of uploading a CSV?

**Comment by mphprogrammer** [comment]
Yes, first release was using the csvs ..., 2nd using any available apis ...

**Comment by jeffmaher** [comment]
Gotcha. Filed an Issue for the CS API: https://github.com/department-of-veterans-affairs/gibct-data-service/issues/21

Otherwise, LGTM. Feel free to merge after tests pass.

**Pull request merged**